flink-connector-opensearch/src/main/java/org/apache/flink/connector/opensearch/sink/OpensearchWriter.java

```diff
@@ -122,10 +128,11 @@
         } catch (Exception e) {
             throw new FlinkRuntimeException("Failed to open the OpensearchEmitter", e);
         }
+        this.failureHandler = failureHandler;
     }
 
     @Override
-    public void write(IN element, Context context) throws IOException, InterruptedException {
+    public void write(IN element, Context context) throws InterruptedException {
```

Comment:
Please revert these changes, they violate the `SinkWriter` interface 

[1] https://github.com/apache/flink/blob/master/flink-core/src/main/java/org/apache/flink/api/connector/sink2/SinkWriter.java#L41

Reply:
@lilyevsky could you please address this and [this](https://github.com/apache/flink-connector-opensearch/pull/11/files#r1110066551) comment? thank you

Reply:
@reta No problem, could you please clarify: you want me to put back the IOException to both write and flush methods, correct? Also, I am not sure what you mean by addressing  [this](https://github.com/apache/flink-connector-opensearch/pull/11/files#r1110066551) . Is it about removing the "@SuppressWarnings("All")"?
Please confirm.

Reply:
@lilyevsky correct, the `throws IOException, InterruptedException` is the right part of the signature: the first comment is about `write`, the second is about `flush`, thank you

Reply:
@reta Done. Also removed the line with warnings suppressions that I added at some point.